### PR TITLE
💄Fix UI Issues

### PR DESCRIPTION
**Pull request by joshbermanssw**
As per #1840 there were a number of formatting issues with different sized views that @bettybondoc wanted changed.

This PR addresses this - ✅ _all test passed by betty_

### CODE REVIEW

File: components/blocks/Features.tsx [hunk 1]
```diff
@@ -7,37 +7,39 @@ import styled from 'styled-components';
...
-          >
-            {data.text}
-          </p>
-          <div className="flex lg:flex-row justify-evenly lg:justify-start">
+          <div className=''>
+          <div className="hidden sm:hidden lg:block lg:ml-0 lg:pl-0 lg:pb-6">
+            <hr className="!my-0 w-full" />
+          </div>
+
+            <p
```
Comment (bradystroud) [inline components/blocks/Features.tsx:35]: Why is there no indenting here?

Do you have a formatter e.g. prettier

https://www.ssw.com.au/rules/consistent-code-style/

File: components/blocks/Features.tsx [hunk 2]
```diff
@@ -7,37 +7,39 @@ import styled from 'styled-components';
...
           )}
-          <hr className="!my-0" />
-          <p
-            className="text-lg lg:text-xl lg:leading-normal block bg-gradient-to-br from-blue-700 via-blue-900 to-blue-1000 bg-clip-text text-transparent -mb-2 max-w-60ch text-balance"
-            data-tina-field={tinaField(data, 'text')}
-          >
-            {data.text}
-          </p>
-          <div className="flex lg:flex-row justify-evenly lg:justify-start">
+          <div className=''>
```
Comment (bradystroud) [inline components/blocks/Features.tsx:30]: Is this div required if there is no classname?

**Review by bradystroud — CHANGES_REQUESTED** [review]

File: components/blocks/Features.tsx (hunk 2 above)
Comment (joshbermanssw) [reply components/blocks/Features.tsx]: removed

File: components/blocks/Features.tsx (hunk 1 above)
Comment (joshbermanssw) [reply components/blocks/Features.tsx]: i forgot to format

File: components/blocks/Features.tsx [hunk 3]
```diff
@@ -1,56 +1,65 @@
...
-import DocsRichText from '../styles/DocsRichText';
-import styled from 'styled-components';
-import playImage from '../../public/img/playButton.png';
+import { Actions } from './Actions'
+import { Container } from './Container'
+import { Prism } from '../styles/Prism'
+import { tinaField } from 'tinacms/dist/react'
+import DocsRichText from '../styles/DocsRichText'
+import styled from 'styled-components'
+import playImage from '../../public/img/playButton.png'
```
Comment (bradystroud) [inline components/blocks/Features.tsx:7]: Not a fan of removing all the ; semicolons... I guess thats a formatting config option.

Is this consistent with the rest of the project?
I think we should make it consistent with the SSW.Website project (new pbi)

**Review by bradystroud — APPROVED** [review]

**Pull request merged**